app/src/main/java/me/hufman/androidautoidrive/music/controllers/SpotifyAppController.kt

```diff
@@ -223,7 +227,7 @@ class SpotifyAppController(context: Context, val remote: SpotifyAppRemote, val w
 				queueItems = emptyList()
 
 				val isLikedSongsPlaylist = playerContext.type == "your_library" || playerContext.type == "your_library_tracks"
-				if (isLikedSongsPlaylist) {
+				if (isLikedSongsPlaylist || playerContext.title == SpotifyWebApi.LIKED_SONGS_PLAYLIST_NAME) {
```

Comment:
I think the `createLikedSongsQueueMetadata` isn't needed once Spotify has switched to playing the private copy of the playlist. There might be the possibility of adding new songs to the public Liked Songs that this function could sync, but I'm not sure the real Spotify even supports this, and you have to reclick Liked Songs to play the new song.
If it is needed here, it's probably better to compare based on LikedSongsState.playlistUri (perhaps memoized) because the translation might change.

Reply:
This is kind of based on this comment I made above (https://github.com/hufman/AndroidAutoIdrive/pull/263#discussion_r622697262) but the idea here is that the if the user tries to load in the dummy Liked Songs playlist it still needs to be handled as though it were the actual Liked Songs playlist in order to promote the sense of unity between the two and not give the user the idea they are two separate entities. If the user did modify the actual Liked Songs playlist then at this point the private copy would be synced during the createLikedSongsQueueMetadata() method which will avoid desync issues between the two as I think we want to have those as much as sync as possible due to the previously mentioned concept. 

I had thought about comparing it based on a dummy playlist uri member variable that would be set from the createLikedSongsQueueMetadata() otherwise if it's null then loading the LikedSongsState.playlistUri from the AppSettings but with that then the Json has to be parsed out if it's not empty and suddenly a good chunk of the work of createLikedSongsQueueMetadata() is being done every time the queueUri changes and a new QueueMetadata needs to be created for that if condition. Then I was thinking the SpotifyWebApi.LIKED_SONGS_PLAYLIST_NAME is not something that would be translated or changed (it shouldn't be) as it's a constant https://github.com/Ben2356/AndroidAutoIdrive/blob/spotifyLikedSongsSkip/app/src/main/java/me/hufman/androidautoidrive/music/spotify/SpotifyWebApi.kt#L39 .

I'm actually not sure on this one since I can see what you mean that the playlistUri is a safer way of determining if the user is opening the dummy Liked Songs playlist but at the same time I'm worried about adding that extra performance overhead as it's something that shouldn't be translated or changed otherwise the user risks breaking it.